.github/workflows/lint.yml

```diff
@@ -17,8 +21,9 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install black~=22.0 pylint numpy pandas
+          python -m pip install black~=22.0 pylint
+          python -m pip install -r requirements.txt
       - name: Autolinting code with black
-        run: black $(git ls-files '*.py')
+        run: black $(git ls-files '*.py') --preview -l 120 -t py311
```

Comment:
What are these changes for? The line number should be set in the config file already. And the target python version should be set already in the matrix strategy.